scavenge/scavenge/go.mod

```diff
@@ -1,33 +1,18 @@
-module github.com/cosmos/sdk-tutorials/scavenge/scavenge
+module github.com/github-username/scavenge
```

Comment:
We should try to keep this `cosmos/sdk-tutorials/scavenge`

Reply:
Why?

Reply:
so it compiles, right?

Reply:
Feel its more natural. What is the reasoning for the change/ Don't consider this a blocker.

Reply:
This is how it gets scaffolded by Starport when the commands are used as outlined in the tutorial. I thought it is easier for the user to compare when things are in the `scavenge` code as created by the tutorial with `starport app github.com/github-username/scavenge --sdk-version="launchpad"`

Reply:
ok ja makes sense to follow the results of the tutorial instructions, also as long as it's consistent it will still compile, right?